### PR TITLE
Added active and marketable filters to CourseRun and Program models

### PR DESCRIPTION
**Pull request by clintonb**
API consumers can now request CourseRuns that are active/marketable and Programs that are marketable.

ECOM-5416

### CODE REVIEW

**Comment by clintonb** [comment]
@bderusha @mjfrey please review

FYI @edx/ecommerce

File: course_discovery/apps/api/v1/views.py [hunk 1]
```diff
@@ -274,6 +272,18 @@ def list(self, request, *args, **kwargs):
...
               paramType: query
               multiple: false
+            - name: active
+              description: Retrieve active course runs
+              required: false
+              type: integer
+              paramType: query
+              multiple: false
+            - name: marketable
+              description: Retrieve marketable course runs
```
Comment (bderusha) [inline course_discovery/apps/api/v1/views.py]: Can we include the definition of "marketable" here and below?

File: course_discovery/apps/api/v1/views.py [hunk 2]
```diff
@@ -274,6 +272,18 @@ def list(self, request, *args, **kwargs):
               type: string
               paramType: query
               multiple: false
+            - name: active
+              description: Retrieve active course runs
```
Comment (bderusha) [inline course_discovery/apps/api/v1/views.py]: can we include our definition of "active" course run here?

**Comment by bderusha** [comment]
One small thing otherwise 👍

**Comment by schenedx** [comment]
👍

**Pull request merged**